sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamAggregationRel.java

```diff
@@ -343,7 +351,11 @@ public void processElement(
             fieldValues.add(windowStartFieldIndex, ((IntervalWindow) window).start());
           }
 
-          o.output(Row.withSchema(outputSchema).addValues(fieldValues).build());
+          Row row =
+              verifyRowValues
+                  ? Row.withSchema(outputSchema).addValues(fieldValues).build()
+                  : Row.withSchema(outputSchema).attachValues(fieldValues);
```

Comment:
I think this is missing a `.build()`

Reply:
pr/10883 changed attachValues() to return a Row, so build() not needed.